src/Service.Host/client/src/components/ChangeRequests/Tabs/BomChangesTab.js

```diff
@@ -41,6 +43,25 @@ function BomChangesTab({ bomChanges, handleSelectChange }) {
         { field: 'changeId', headerName: 'Id', width: 100 }
     ];
 
+    const [weekStartDate, setWeekStartDate] = useState(null);
+
+    const ValidPhaseInWeek = weekStart => {
```

Comment:
function name coukld be camelCase to avoid confusing with a functional React component